tests/functional/conftest.py

```diff
@@ -6,11 +6,7 @@
 
 from report import db
 
-TEST_SETTINGS = {
-    "database_url": os.environ.get(
-        "TEST_DATABASE_URL", "postgresql://postgres@localhost:5436/report_functests"
-    ),
-}
+TEST_SETTINGS = {"database_url": os.environ["TEST_DATABASE_URL"]}
```

Comment:
With the cookiecutter `tox.ini` ensures that `TEST_DATABASE_URL` is always set when the tests run, so the code can just assume it's present